docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,64 +4,109 @@ title: "Red Hat OpenShift"
 
-| OpenShift version | Supported          |
-| ----------------- | ------------------ |
-| 4.12.x            | :white_check_mark: |
-| 4.13.x            | :white_check_mark: |
-| 4.14.x            | :white_check_mark: |
+## Cluster Specification
 
-Any version not explicitly marked in the table above is not tested, and we cannot guarantee compatibility.
+When deploying Camunda 8 on an OpenShift cluster, the cluster specification should align with your specific requirements and workload characteristics. Here's a suggested configuration to begin with:
 
-## Pitfalls to avoid
+- **Instance type:** 4 vCPUs (x86_64, >3.1 GHz), 16 GiB Memory (for example, [m5.xlarge on AWS](https://aws.amazon.com/en/ebs/general-purpose/))
+- **Number of dedicated nodes:** 4
+- **Volume type:** SSD volumes (with between 1000 and 3000 IOPS per volume, and a throughput of 1,000 MB/s per volume, for instance, [gp3 on AWS](https://aws.amazon.com/en/ebs/general-purpose/))
 
-For general deployment pitfalls, visit the [deployment troubleshooting guide](/self-managed/operational-guides/troubleshooting/troubleshooting.md).
+We conduct testing against the following OpenShift versions and ensure compatibility with versions supported by Red Hat, following [Red Hat's OpenShift Lifecycle Policy](https://access.redhat.com/support/policy/updates/openshift), which typically encompasses the last four versions:
 
-### Security Context Constraints (SCCs)
+| OpenShift Version | [End of Support Date](https://access.redhat.com/support/policy/updates/openshift) |
+| ----------------- | --------------------------------------------------------------------------------- |
+| 4.15.x            | August 27, 2025                                                                   |
+| 4.14.x            | May 1, 2025                                                                       |
+| 4.13.x            | November 17, 2024                                                                 |
+| 4.12.x            | July 17, 2024                                                                     |
 
-Much like how roles control the permissions of users, Security Context Constraints (SCCs) are a way to control the permissions of the applications deployed, both at the pod and container level. It's generally recommended deploying your application with the most restricted SCCs possible. If you're not familiar with security context constraints, refer to the [OpenShift documentation](https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html).
+Please note that any version not explicitly listed in the table above hasn't been tested, and we cannot guarantee compatibility.
```

Comment:
> Please note that the change in the versions referenced as supported in the documentation is related to the versions we are now testing, linked to https://github.com/camunda/camunda-platform-helm/pull/1816. For versions 4.12 and 4.13, support ends according to https://access.redhat.com/support/policy/updates/openshift on July 17, 2024 (Maintenance support ends) for 4.12 and November 17, 2024 for 4.13 : we do not have tested them since a while but we continue to support them until the indicated EOL.

Thank you for the additional context regarding OpenShift versioning. I prefer we get to a point where we have more definitive language around what we test and support. Is it OpenShift's end of support date that is driving our compatibility guarantee? Is it the last four versions or the generally available versions? 

Can we make line 31 briefer? "Note that we cannot guarantee compatibility for any version not explicitly listed in the table." It feels implied that we test if we guarantee compatibility.

Reply:
Thank you @leiicamundi! @Langleu touches on a similar thought around compatibility guarantees in a [separate conversation](https://github.com/camunda/camunda-docs/pull/3873#discussion_r1627876868).

@conceptualshark I know you are still super early in your onboarding, but can you give this PR a read? Focusing on this section.

Reply:
I'm sorry. I didn't see that there was already an open topic, but agree I'm not 100% sure how it aligns with our maintenance guarantees.

Reply:
Based on recent changes to the PR, I will resolve this conversation. The new content and structure is clearer and we can always modify if we see customers interpreting this in _unique_ ways.